### PR TITLE
added scan function in the resource.py and fixed typo

### PR DESCRIPTION
**Pull request by jtfrench23**
I added a scan function so that users can check the entries in the database they created.  This will add confidence to the user that they have been successful in setting up the db.  Let me know if you think it is worth adding some information on NoSQL databases and when to use them in this example, that may be unnecessary information for the example and we can assume that if a person is using this example they already know about NoSQL and DynamoDB otherwise they would probably be looking at the relationalDB example instead.  Just give me your thoughts and I can move forward from there.

### CODE REVIEW

**Review by Pia007 — CHANGES_REQUESTED** [review]
Hi Jordan, it looks good. Left one minor change. I would have to test out the example. Thanks for the explaining your rationale for the scan.

File: content/docs/examples/dynamodb.md [hunk 1]
```diff
@@ -28,11 +28,16 @@ Deploy the resources.
 ```bash
 cdev deploy
 ```
-Exectute the function.
+Execute the function.
 ```bash
 cdev run function.execute hello_world_comp.email_adder --event-data "{\"body\":{\"first_name\":\"Paul\",\"last_name\":\"Atreides\",\"email\":\"Muaddib@dune.com\"}}"
 ```
-Check the logs from the function
+You can run the function above a few times after making changes to the email address (and other fields if you wish, but the email is mandatory as it is the primary key), to add more entries to your database and then excute the scan function with the command below:
```
[the remark this answers is not in the document]
Comment (jtfrench23) [reply content/docs/examples/dynamodb.md:35]: good catch

**Review by Pia007 — APPROVED** [review]
Looks good, Jordan!

**Review by ianick87 — APPROVED** [review]

**Review by DanielSanchez97 — APPROVED** [review]
LGTM

**Pull request merged**